codalab/worker/worker.py

```diff
@@ -260,11 +260,16 @@ def check_num_runs_stop(self):
 
     def start(self):
         """Return whether we ran anything."""
-        self.load_state()
-        self.sync_state()
-        self.image_manager.start()
-        if not self.shared_file_system:
-            self.dependency_manager.start()
+        try:
```

Comment:
Would it be harmful to run `cleanup()` at, say, the line where we call `self.load_state()` (because the image manager hasn't been started yet)? @teetone , what do you think?